source/server/admin/stats_handler.cc

```diff
@@ -179,7 +188,9 @@ Http::Code StatsHandler::handlerContention(absl::string_view,
 void StatsHandler::statsAsText(const std::map<std::string, uint64_t>& all_stats,
                                const std::map<std::string, std::string>& text_readouts,
                                const std::vector<Stats::ParentHistogramSharedPtr>& histograms,
-                               bool used_only, const absl::optional<std::regex>& regex,
+                               bool used_only,
+                               Utility::HistogramBucketsValue histogram_buckets_value,
```

Comment:
s/Value/Mode/
s/_value/mode/

etc. That would be more consistent with similar enums in the stats world.

Reply:
Renamed value to mode.